src/java/picard/sam/MergeSamFiles.java

```diff
@@ -83,6 +90,9 @@
     @Option(doc = "Comment(s) to include in the merged output file's header.", optional = true, shortName = "CO")
     public List<String> COMMENT = new ArrayList<String>();
 
+    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the locations of the positions to merge. Assume bam are sorted and indexed.", optional = true)
```

Comment:
Please comment on how this should be used. In particular I'm worried of people using this to scatter merge bam alignment and the resulting mess that would arise from that (since a read can overlaps more than one locus...)